code42_connector.py

```diff
@@ -65,19 +74,45 @@ def _handle_test_connectivity(self, param):
             # Ideally the client instantiation would happen in `initialize()` but that function does not have access
             # to the action, so it cannot effectively report errors to the UI.
             # Fix this with a decorator or something similar.
-            self._client = py42.sdk.from_local_account(self._cloud_instance, self._username, self._password)
-            self._client.users.get_current()
+            self.client.users.get_current()
         except Exception as exception:
             return action_result.set_status(phantom.APP_ERROR, f"Unable to connect to Code42: {str(exception)}")
 
         self.save_progress("Test Connectivity Passed")
         return action_result.set_status(phantom.APP_SUCCESS)
 
+    def _handle_add_departing_employee(self, param):
+        self.save_progress("In action handler for: {0}".format(self.get_action_identifier()))
+        action_result = self.add_action_result(ActionResult(dict(param)))
+        username = param["username"]
+        departure_date = param.get("departure_date")
+        user_id = self._get_user_id(username)
+        self.client.detectionlists.departing_employee.add(user_id, departure_date=departure_date)
+        action_result.update_summary({})
```

Comment:
I think this and/or `set_summary` are basically the equivalents of the table outputs in XSOAR. It might make sense to choose a couple relevant properties for each of these actions and add them here

Reply:
Like the parameters?
I don't think there are responses for these API calls

Reply:
no, but you could still do something like output the ID and username of the user that was added / removed / etc. Isn't that what we do in XSOAR?

Reply:
Oh okay I see, yeah let me take a closer look at this and probably copy XSOAR

Reply:
`add_data` would definitely be important to test since if that were to change it could break Playbooks and such.
Since `summary` is just visual that is probably less critical.

Reply:
It does seem like we might get a lot of this for free, like the action's parameters used seem to automatically go to outputs under a sub-obj `parameters`, as well as the status of the action etc.